opentelemetry-sdk/tests/metrics/test_metrics.py

```diff
@@ -60,17 +64,28 @@ def test_collect_no_metrics(self):
         meter.collect()
         self.assertFalse(batcher_mock.process.called)
 
+    def test_collect_not_registered(self):
+        meter = metrics.MeterProvider().get_meter(__name__)
+        batcher_mock = mock.Mock()
+        meter.batcher = batcher_mock
+        counter = metrics.Counter(
+            "name", "desc", "unit", float, meter
+        )
+        labels = {"key1": "value1"}
+        counter.add(1.0, labels)
+        meter.collect()
+        self.assertFalse(batcher_mock.process.called)
```

Comment:
Even if no view is registered, the default aggregator with all keys must be applied right? Or this decision has not been finalized?

Reply:
Yes you are correct. I did not fix the tests to reflect that yet.